corehq/apps/sms/api.py

```diff
@@ -252,6 +255,7 @@ def send_message_via_backend(msg, backend=None, orig_phone_number=None):
     orig_phone_number - the originating phone number to use when sending; this
       is sent in if the backend supports load balancing
     """
+    sms_load_counter("outbound", msg.domain)()
```

Comment:
do we want to track this even if the messages isn't sent (e.g. domain doesn't have permissions)?

Reply:
I think that should be a pretty rare scenario, but this is a good point. I'll move it.

Reply:
As I'm thinking about this more I'm a little conflicted. This load represents more than just "a message was sent"—it also includes resources consumed by moving the message through the queue, etc. We will have no indication in these stats if there are a large number of messages moving through the system that are ultimately discarded due to issues like missing permissions (although that scenario would be unexpected). Assuming there are only a few messages that get disqualified due to lack of permissions, it will not make a big difference to track them here.